crates/fuel-gas-price-algorithm/gas-price-analysis/src/simulation.rs

```diff
@@ -180,7 +190,7 @@ fn fullness_and_bytes_per_block(size: usize, capacity: u64) -> Vec<(u64, u64)> {
     let bytes_scale: Vec<_> = std::iter::repeat(())
         .take(size)
         .map(|_| rng.gen_range(0.5..1.0))
-        .map(|x| x * 4.0)
+        .map(|x| x * 0.01)
```

Comment:
```suggestion
        .map(|x| x * BYTES_SCALE_INC)
```
it would be nice to avoid use of magic numbers here